frontend/src/Components/ForgotPassword/ForgotPassword.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useRef, useState } from "react";
+import ROUTES from "../Routes/ROUTES";
+import { Link } from "react-router-dom";
+import CenteredContainer from "../ReusableComponents/CenteredContainer";
+import AxiosInstance from "../axios/AxiosInstance";
+
+const ForgotPassword = () => {
+   const [email, setEmail] = useState("");
+   const emailRef = useRef();
+
+   const onEmailChange = (e) => {
+      setEmail(e.target.value);
+   };
+
+   const onFormSubmit = (e) => {
+      e.preventDefault();
+      onSuccessfulChange();
+   };
+
+   const onSuccessfulChange = async () => {
+      try {
+         await AxiosInstance.post("/users/password-reset/request", { email });
+         console.log("SUCCESS");
```

Comment:
Console.logs are okay for developing, but should be avoided to be pushed on the main.